spec/controllers/redirects_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe RedirectsController do
+  describe 'auth' do
+    context 'when not logged in' do
+      it 'redirects user to login' do
+        get :new
+        expect(response).to_not be_success
+      end
+    end
+  end
+end
+
+describe RedirectsController do
+  let(:current_user) { create(:user) }
+
+  before do
+    create(:site)
+    sign_in current_user
+  end
+
+  describe '#new' do
+    it 'works' do
+      get :new
+      expect(response).to be_success
+    end
```

Comment:
Likewise, this doesn't really test what this action is doing. There's no test to assert that the assign is happening or that the new template is being rendered.